Makefile

```diff
@@ -72,6 +75,9 @@ artifacts: clean
 	GOOS=linux GOARCH=s390x       make -C $(CURDIR) binaries
 	tar $(TAR_FLAGS) -czvf $(CURDIR)/_output/nerdctl-$(VERSION_TRIMMED)-linux-s390x.tar.gz   _output/nerdctl extras/rootless/*
 
+	GOOS=windows GOARCH=amd64       make -C $(CURDIR) binaries
+	tar $(TAR_FLAGS) -czvf $(CURDIR)/_output/nerdctl-$(VERSION_TRIMMED)-windows-amd64.tar.gz   _output/nerdctl extras/rootless/*
```

Comment:
Windows users may prefer zip over tar.gz? idk

Reply:
`extras/rootless` is useless for Windows

Reply:
We have `tar` on windows these days.  containerd uses tar.gz and i think it makes sense match that

Reply:
removed `extras/rootless`